docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -358,12 +358,12 @@ constexpr D3D11_INPUT_ELEMENT_DESC vertexInputLayoutInfo[] =
 
 Now let's make sense of what each field means, let's go over one by one in order.
 
-- `SemanticName`: let's us refer to a particular field given the string after the colon in HLSL (remember `float3 position: POSITION`)
+- `SemanticName`: lets us refer to a particular field given the string after the colon in HLSL (remember `float3 position: POSITION`)
```

Comment:
`lets us` should have been `let us` right? or just `let's` :)

Reply:
"let's" = "let us", so it effectively said "let us us"
"lets" without the apostrophe is a different word and is the one we actually want here because its the conjucated version of "letting": "It lets us [...]".

Reply:
ah i'm blind :)